integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -90,11 +93,12 @@ def _create_group_webhook(
             f"Creating webhook for {group.get_id()} with events: {[event for event in webhook_events if webhook_events[event]]}"
         )
         try:
-            resp = group.hooks.create(
+            resp = await AsyncFetcher.fetch_single(
```

Comment:
I feel like maybe `fetch_single` isn't the appropriate name for it

Reply:
Yeah I felt the same but I was also worried about duplication since the implementation will look the same, only the name of the method will change